Makefile

```diff
@@ -6,13 +6,16 @@ init:
 	pip install -e '.[dev]'
 
 test:
-	pytest --cov samtranslator --cov-report term-missing --cov-fail-under 95 tests
+	pytest --cov samtranslator --cov-report term-missing --cov-fail-under 95 tests/*
+
+test-integ:
+	pytest --no-cov tests_integ/*
```

Comment:
Same nit here: I would prefer if we standardize our naming and structure across all our repos we own.

Reply:
Just so I understand correctly, does this mean moving the whole content of the current `tests` dir under `tests\unit` and moving the integration tests under `tests\integration`?

Reply:
Just moving the `tests_integration` content in `tests/integration` will make the unit tests also run the integration tests so we may have to move everything.